connector/oidc/oidc.go

```diff
@@ -215,10 +225,57 @@ func (c *oidcConnector) HandleCallback(s connector.Scopes, r *http.Request) (ide
 		Email:         claims.Email,
 		EmailVerified: claims.EmailVerified,
 	}
+
+	// Add AccessToken to user identity for future requests
+	connData, err := json.Marshal(connectorData{
```

Comment:
Connector data is private, containing data like upstream access tokens. It shouldn't be shared.

Reply:
The AccessToken can still be encrypted as per your suggestion: https://github.com/coreos/dex/issues/376#issuecomment-271811004

Reply:
If it's okay I'm just going to submit an implementation.

Reply:
Yes, of course.